source/game.programmeproducer.bmx

```diff
@@ -416,12 +417,15 @@ Type TProgrammeProducer Extends TProgrammeProducerBase
 			                         ]
 			Local feeCurrent:Int = current.GetJobBaseFee(jobId, 1, -1)
 			Local feeAlternative:Int = alternative.GetJobBaseFee(jobId, 1, -1)
-			Local sumCurrent:Float = xpCurrent
-			Local sumAlternative:Float = xpAlternative
-			'TODO genre fit; job may also not fit (due to too little alternatives)
+			Local sumCurrent:Float = 2 * xpCurrent
```

Comment:
Wie kommen die Multiplikatoren (2 und 4) zustande? Was bewirkt die erhoehte Summe "effektiv" ?

Reply:
Für die Bewertung einer Besetzung summiere ich einfach alle Attribute. Je höher, desto besser. Manche Attribute sind aber wichtiger als andere (die, die laut Genre-Definition einen Wert größer 0 bekommen). Die gehen (der Einfachheit/aus Bauchgefühl) mehrfach in die Wertung ein. Dadurch soll erreicht werden, dass bei einem Actionfilm jemand mit viel Power trotzdem besser dasteht als jemand ohne Power aber sonst höheren Werten.
Die Erfahrung bewerte ich weniger hoch, da die ja im Laufe der Zeit anwächst. (Die anderen Attribute sollen "gefördert" werden.)